src/Factories/FunFunFactory.php

```diff
@@ -957,12 +957,18 @@ public function newErrorPageHtmlPresenter(): ErrorPageHtmlPresenter {
 		return new ErrorPageHtmlPresenter( $this->getLayoutTemplate( 'Error_Page.html.twig' ) );
 	}
 
-	public function newInternalErrorHtmlPresenter(): InternalErrorHtmlPresenter {
-		return new InternalErrorHtmlPresenter( $this->getLayoutTemplate( 'Error_Page.html.twig' ) );
+	public function getInternalErrorHtmlPresenter(): ExceptionHtmlPresenterInterface {
+		$presenter = $this->pimple['internal_error_html_presenter'];
```

Comment:
Please use the `sharedObject` property and the `createSharedObject` function, pimple is on its way out. That way, you can also have a default presenter and leave the `setErrorPageHtmlPresenter` method empyt for the prod  and test environments.